effekt/shared/src/main/scala/effekt/util/Timer.scala

```diff
@@ -17,9 +17,19 @@ trait Timers {
   val times: mutable.LinkedHashMap[String, mutable.ListBuffer[Timed]] = mutable.LinkedHashMap.empty
 
   /** Whether the `timed` function is NOP or actual takes and saves the time. */
-  var timersActive: Boolean
+  var timersActive: Boolean = true
```

Comment:
Before the default was `false`. Is this relevant?

Reply:
I figured, if someone forgets to call `clearTimers`, they'd still expect the timers to work. Since deactivating the timers, i.e., turning `timed` into a NOP, is a specific optimisation, it should not be the default.